ibr/ibr-sdk/src/renderers.js

```diff
@@ -75,18 +75,21 @@ function init(ibrRawData) {
  * Enable Export and parse top level of decoded IBR Object into structures.
  * @param {IBRObject} ibrObject IBRObject created from input IBR binary data
  file.
- * @param {number} structureIndex Index of current structure(floor).
  * @param {HTMLElement} parentElement parent HTML element that the
   visualization will be append on.
+ * @param {Map.<String, List<Mesh>>} spaceLib Space library stores space
+ name and mesh object(s).
+ * @param {Map.<String, List.<String, Line>>} connectionLib Boundary name and
+ corresponding list of neighbor names and three.js Line object.
  * @return {Object} scene The Scene Object that all THREE objects are
  rendered on.
  */
-function render(ibrObject, structureIndex, parentElement) {
+function render(ibrObject, parentElement, spaceLib, connectionLib) {
   const scene = generateScene(parentElement);
   document.getElementById('dwn-btn').style.display = 'block';
   document.getElementById('filename').style.display = 'block';
-  // Visualization visualizations of current structure
-  renderVisualizations(ibrObject, structureIndex, scene);
+  // Render current structure
+  renderSingleIBRStructure(ibrObject, 0, scene, spaceLib, connectionLib);
```

Comment:
i noticed you got rid of 0 in the other render function, let's try to resolve this one soon as well :)